src/pages/Group.jsx

```diff
@@ -0,0 +1,134 @@
+import { Avatar, Typography, Divider } from "@mui/material";
+import { styled } from "@mui/material/styles";
+// import CakeTwoToneIcon from '@mui/icons-material/CakeTwoToneIcon';
+
+/* custom styled components */
+const GroupContainer = styled("div")`
+  border-radius: 30px;
+  background-color: white;
+  margin: 15px;
+  padding: 5px;
+  .MuiTypography-root {
+    font-family: "Nunito Sans", sans-serif;
+  }
+`;
+
+const GroupDetails = styled("div")`
+  display: flex;
+  padding: 10px;
+  margin: 10px;
+  justify-content: space-between;
+  border-bottom: 0.5px ridge #d5d0d0;
+  .group-details {
+    display: flex;
+    flex: 1;
+  }
+  .money {
+    font-weight: 500;
+    align-content: center;
+  }
+  .date {
+    font-size: 12px;
+    color: #7a7a7a;
+    font-weight: 700;
+  }
+`;
+
+const GroupAvatar = styled(Avatar)`
+  width: ${(props) => props.width};
+  height: ${(props) => props.height};
+  margin-right: 10px;
+`;
+
+const Balances = styled("div")`
+  display: flex;
+  flex-direction: column;
+`;
+
+const Debt = styled("div")`
+  display: flex;
+  padding: 8px 10px;
+
+  .debt-text {
+    font-size: 15px;
+    color: #7a7a7a;
+    // font-weight: 700;
```

Comment:
remove?

Reply:
you meant the styles to be placed in another js file?

Reply:
the comment